include/ddc/discrete_domain.hpp

```diff
@@ -187,6 +193,33 @@ class DiscreteDomain
                 DiscreteVector<DDims...>((get_or<DDims>(oextents, get<DDims>(myextents)))...));
     }
 
+    template <class... DElems>
+    bool is_inside(DElems const&... delems) const noexcept
+    {
```

Comment:
is_inside(...) is now available in both DiscreteDomain and StridedDiscreteDomain. Is the function a feature request or is it useful for implementing generalized domains?